spec/features/user_adds_a_study_spot_spec0.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+feature "adding spot" do
+  scenario "user adds a new spot" do
+
+    visit new_user_registration_path
+
+    fill_in 'Email', with: 'john@example.com'
+    fill_in 'Password', with: 'password'
+    fill_in 'Password confirmation', with: 'password'
+    fill_in 'First name', with: 'John'
+    fill_in 'Last name', with: 'Example'
+    fill_in 'Username', with: 'johnex'
```

Comment:
Same comment as above - go through the steps to sign in an already-created user, rather than register a totally new one